validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -223,7 +235,8 @@ public static ValidatorLoader create(
       final PublicKeyLoader publicKeyLoader,
       final AsyncRunner asyncRunner,
       final MetricsSystem metricsSystem,
-      final Optional<DataDirLayout> maybeMutableDir) {
+      final Optional<DataDirLayout> maybeMutableDir,
+      final boolean allowNoLoadedKeys) {
```

Comment:
The value of `allowNoLoadedKeys` exists within `ValidatorConfig`. There is no reason to add it as an extra parameter on create(). You can just use `config.allowNoLoadedKeys()` when needed.